Storage/FirebaseStorageUI/NSURL+SDWebImageFIRStorageLoader.h

```diff
@@ -0,0 +1,38 @@
+//
+//  Copyright (c) 2016 Google Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+#import <Foundation/Foundation.h>
+#import <FirebaseStorage/FirebaseStorage.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface NSURL (SDWebImageFIRStorageLoader)
```

Comment:
Remember to change this category when renaming classes: `FUIStorageLoader`

Reply:
What about `FUIStorageImageLoader` ? Or is there are any plan, for support non-image resource ?

Reply:
You're right, `FUIStorageImageLoader` is better 👍